<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro_cliente</class>
 <widget class="QWidget" name="Registro_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ventana_registroclientes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1024</width>
     <height>791</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Recursos/images.qrc">:/operaciones-design-blanco.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="id_clientes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_productos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro de productos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_proveedores">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro de proveedores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_lineaproductos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Linea de productos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_trabajadores">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro de trabajadores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_regresaroperaciones">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
   <property name="icon">
    <iconset resource="../Recursos/images.qrc">
     <normaloff>:/regreso.png</normaloff>:/regreso.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos/images.qrc"/>
 </resources>
 <connections/>
</ui>
